webserver/courses/views.py

```diff
@@ -122,3 +124,62 @@ def get_course_info(request, id):
             ),
             status=400,
         )
+
+@api_view(['POST'])
+def add_image_to_course(request, id):
+    if request.user.is_anonymous:
+        return Response(
+            data=dict(
+                reason='You must be logged in to join a course.',
+            ),
+            status=400,
+        )
+    if request.FILES.get('image') == None:
+        return Response(
+            data=dict(
+                reason='Image file is required.',
+            ),
+            status=400,
+    )
```

Comment:
Need to add more validation here to make sure valid images are being uploaded:

- Use `json.loads()` to parse the json, if it's unparsable return 400 response stating that the file content must be valid
- Check the `version` field on the json and ensure it's [2, 0, 0], return a message saying the file was created using an incompatible version of the labeling tool if it's not
  - I added that field so I could increment it when substantial or breaking changes get made to the labeling tool export format